api/core/indexing_runner.py

```diff
@@ -78,8 +78,8 @@ def run(self, dataset_documents: list[DatasetDocument]):
                     dataset_document=dataset_document,
                     documents=documents,
                 )
-            except DocumentIsPausedException:
-                raise DocumentIsPausedException("Document paused, document id: {}".format(dataset_document.id))
+            except DocumentIsPausedExceptionError:
```

Comment:
The same error

Reply:
We now have no occurance of `ExceptionError` appendix in class names.